src/renderer/components/common/QrCode/QrReader/QrReader.tsx

```diff
@@ -182,9 +193,14 @@ const QrReader = ({
         continue;
       }
 
-      const result = EXPORT_ADDRESS.decode(fountainResult.slice(3));
+      let result;
+      try {
+        result = EXPORT_ADDRESS.decode(fountainResult.slice(3));
+      } catch (e) {
+        result = EXPORT_ADDRESS_2.decode(fountainResult.slice(3));
+      }
```

Comment:
This bit repeats two times

Reply:
this is made for 1st and next frames handling. In the 1st frame the app is able to understand which kind of QR is present: with the string or Raptor encoded data structure.
In the 2nd and next frames handler the application parses the QR code and accumulates Raptor frames.

Reply:
Yea I meant we can extract this bit into a separate function

Reply:
P.S. I see that this code was not replaced with just EXPORT_ADDRESS.decode